cpp/src/io/parquet/writer_impl.cu

```diff
@@ -32,7 +33,9 @@
 #include <cudf/null_mask.hpp>
 #include <cudf/strings/strings_column_view.hpp>
 #include <cudf/table/table_device_view.cuh>
+#include <cudf/utilities/span.hpp>
 
+#include <iostream>
```

Comment:
Looks like this was left in from some print debugging? If it is needed, it should go with the other section of stdlib headers like `#include <algorithm>` below here, rather than with the rmm includes.